src/createServer.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+
+const userServices = require('./services/users');
+const expenseServices = require('./services/expenses');
+const { userRouter } = require('./routes/users');
+const { expenseRouter } = require('./routes/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  userServices.removeAllUsers();
+  expenseServices.removeAllExpenses();
+
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expenseRouter);
```

Comment:
move express.json() as separate common middleware

Reply:
thank you for review!